css/styles.css

```diff
@@ -209,6 +309,16 @@ li > p {
   justify-content: space-between;
 }
 
+.firstElement {
+  display: none;
+}
+
+@media screen and (min-width: 992px) {
+  .firstElement {
+    display: flex;
+  }
+}
+
```

Comment:
- [ ] Kindly group  the same breakpoints  together check the comment above arrow ⬆️